src/app/domains/Plan/routes/PlanCreate/PlanCreate.layout.js

```diff
@@ -1,50 +1,179 @@
  *
  * @comment PlanCreate - React component.
  *
- * @since 05 Mar 2021 ( v.0.0.1 ) // LAST-EDIT DATE
+ * @since 14 Mar 2021 ( v.0.0.2 ) // LAST-EDIT DATE
  *
  * @return {ReactComponent}
  */
 
-const PlanCreate = (props) => {
-  // [INTERFACES]
-  /*
-  code sample: 
-  const { data } = props
-  */
+const MOCK_DATA = [
+  {
+    title: 'Software design',
+    key: uuidv4(),
+    children: [
+      {
+        title: 'Angular developer',
+        key: uuidv4(),
+        children: [
+          { title: 'JavaScript', key: uuidv4() },
+          { title: 'Angular', key: uuidv4() },
+          { title: 'TypeScript', key: uuidv4() },
+          { title: 'HTML', key: uuidv4() },
+          { title: 'CSS', key: uuidv4() }
+        ]
+      },
+      {
+        title: 'React developer',
+        key: uuidv4(),
+        children: [
+          { title: 'JavaScript', key: uuidv4() },
+          { title: 'React', key: uuidv4() },
+          { title: 'CSS', key: uuidv4() }
+        ]
+      },
+      {
+        title: 'Backend developer',
+        key: uuidv4(),
+        children: [
+          { title: 'JavaScript', key: uuidv4() },
+          { title: 'HTML', key: uuidv4() },
+          { title: 'Node.js', key: uuidv4() },
+          { title: 'Express.js', key: uuidv4() }
+        ]
+      }
+    ]
+  },
+  {
+    title: 'User experience design',
+    key: uuidv4(),
+    children: [
+      {
+        title: 'UI/UX Designer',
+        key: uuidv4(),
+        children: [
+          { title: 'Figma', key: uuidv4() },
+          { title: 'Color palette', key: uuidv4() },
+          { title: 'SWG animation', key: uuidv4() },
+          { title: 'CSS', key: uuidv4() },
+          { title: 'HTML', key: uuidv4() }
+        ]
+      }
+    ]
+  },
+  {
+    title: 'Innovation',
+    key: uuidv4(),
+    children: [{ title: 'Management', key: uuidv4() }]
+  }
+]
 
+const PlanCreate = () => {
   // [ADDITIONAL_HOOKS]
-  /*
-  code sample: 
-  const firestore = useFirestore()
-  */
+  const history = useHistory()
 
   // [COMPONENT_STATE_HOOKS]
-  /*
-  code sample:
-  const singleton = useRef(true) // references also put here
-  const [state, setState] = useState({})
-  */
+  const [technologies, setTechnologies] = useState([])
+  const [isOverview, setIsOverview] = useState(false)
 
   // [HELPER_FUNCTIONS]
+  const onNext = () => console.log('asd')
+  const onCheck = (item, { checkedNodes }) => {
+    const filteredTech = checkedNodes.filter(
+      (technology) => !technology.hasOwnProperty('children')
+    )
 
-  // [COMPUTED_PROPERTIES]
-  /* 
-    code sample: 
-    const userDisplayName = user.firstName + user.lastName
-  */
-
-  // [USE_EFFECTS]
+    setTechnologies(_.uniqBy(filteredTech, 'title'))
+  }
 
   // [TEMPLATE]
-  return <div>PlanCreate</div>
-}
+  return (
+    <>
+      {!isOverview ? (
+        <>
+          <Content bg="#ffffff" paddingTop={4} paddingX={4}>
+            <PageWrapper
+              title="Create plan for Dima"
```

Comment:
you need to pass a prop from parent component to title, (name instead 'Dima') maybe